Tests/IGListStackSectionControllerTests.m

```diff
@@ -416,6 +416,26 @@ - (void)test_whenQueryingVisibleSectionControllers_withCellsOffscreen_thatOnlyVi
     XCTAssertEqual([stack visibleCellsForSectionController:section5].count, 0);
 }
 
+- (void)test_whenQueryingVisibleSectionControllers_withIndexPathsOffscreen_thatOnlyVisibleReturned {
+    [self setupWithObjects:@[
+                             [[IGTestObject alloc] initWithKey:@0 value:@[@3, @4, @0, @5, @6]]
+                             ]];
+    IGListStackedSectionController *stack = [self.adapter sectionControllerForObject:self.dataSource.objects[0]];
+    
+    IGListTestSection *section1 = stack.sectionControllers[0];
+    IGListTestSection *section2 = stack.sectionControllers[1];
+    IGListTestSection *section3 = stack.sectionControllers[2];
+    IGListTestSection *section4 = stack.sectionControllers[3];
+    IGListTestSection *section5 = stack.sectionControllers[4];
+    
+    XCTAssertEqual([self.adapter visibleIndexPathsForSectionController:stack].count, 10);
+    XCTAssertEqual([stack visibleIndexPathsForSectionController:section1].count, 3);
+    XCTAssertEqual([stack visibleIndexPathsForSectionController:section2].count, 4);
+    XCTAssertEqual([stack visibleIndexPathsForSectionController:section3].count, 0);
+    XCTAssertEqual([stack visibleIndexPathsForSectionController:section4].count, 3);
+    XCTAssertEqual([stack visibleIndexPathsForSectionController:section5].count, 0);
```

Comment:
Last request, promise!

Can we check the actual values here and not just the counts? That'll be a really accurate test! It'll be a bit long-winded, but a more thorough test will withstand changes better.

Here's what I'm thinking:

```objc
// use sets b/c we don't care about order
NSSet *visible1 = [NSSet setWithArray:[stack visibleIndexPathsForSectionController:stack.sectionControllers[0]]];
// repeat for all 5

NSSet *expected1 = [NSSet setWithArray:@[
  [NSIndexPath indexPathForItem:0 inSection:0],
  // etc
]];
// repeat for all 5

XCTAssertEqualObjects(visible1, expected1);
// repeat for all 5
```

Make sense?